CONTRIBUTING.md

```diff
@@ -29,6 +29,69 @@ Documentation for the in-the-wild deployments lives in the
 
 See [DEVELOP.md](./DEVELOP.md) for more information on how to run the development environment.
 
+## Proposing Changes
+The Wormhole project understands that contributions come in all shapes and sizes and from both from casual contributions (eg. fix a typo) to advanced contributions (eg. add a new chain integration).  Understanding this, the Wormhole project aims to offer the following paths to make contributions to Wormhole.
+### GitHub Pull Request (aka: Casual Contribution)
+**Objective:** Allow any casual contribution to the project using the default tooling provided by GitHub.
+
+**Examples of casual contribution**
+- Adjustments to documentation
+- Adjustments to configuration parameters
+- Fixing a script or otherwise non-mission critical piece of code
+
+**Process**
+
+1. Fork the Wormhole project ([GitHub Official Documentation](https://docs.github.com/en/get-started/quickstart/fork-a-repo)).
+1. Clone the forked repository ([GitHub Official Documentation](https://docs.github.com/en/repositories/creating-and-managing-repositories/cloning-a-repository#cloning-a-repository))
+1. Change directory into the Wormhole repository
+    - `cd ./wormhole`
+1. Create a new branch
+    - `git checkout -b NAME_OF_FEATURE_BRANCH`
+1. Make changes to relevant files
+1. Add your changes
+      - `git add FILE_YOU_CHANGED`
+1. Commit your changes
+      - `git commit -m "docs: updated contributing process"`
+      - *See below for commit message convention guidance*
+1. Push your changes to GitHub
+      - `git push origin NAME_OF_FEATURE_BRANCH"`
```

Comment:
Might make sense to have something in this process list about what type of rebases the user could/should select if the branch gets out of sync from dev.v2?

Reply:
I think if we're going to go with the approach where github PRs get mirrored into gerrit by a bot, then I think it doesn't matter what the author does.  The bot will just upload the merge/rebase as a new patch set on gerrit.